New jitterentropy library broke configure. This patch fixes it.

Upstream patch: https://github.com/nhorman/rng-tools/commit/daff4e2dd9d3abb74829f41e72607681163bd309

--- a/configure.ac	2020-12-03 20:59:11.086238518 +0100
+++ b/configure.ac	2020-12-03 21:00:40.978593727 +0100
@@ -91,7 +91,7 @@
 		AC_SEARCH_LIBS(jent_version,jitterentropy,
 				[AM_CONDITIONAL([JITTER], [true])
 				AC_DEFINE([HAVE_JITTER],1,[Enable JITTER])],
-				AC_MSG_NOTICE([No Jitterentropy library found]))
+				AC_MSG_NOTICE([No Jitterentropy library found]),-lpthread)
 	], [AC_MSG_NOTICE([Disabling JITTER entropy source])]
 )
 
